lib/ansible/modules/remote_management/oneview/oneview_logical_interconnect_group.py

```diff
@@ -0,0 +1,167 @@
+#!/usr/bin/python
+# Copyright (c) 2016-2017 Hewlett Packard Enterprise Development LP
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+DOCUMENTATION = '''
+---
+module: oneview_logical_interconnect_group
+short_description: Manage OneView Logical Interconnect Group resources
+description:
+    - Provides an interface to manage Logical Interconnect Group resources. Can create, update, or delete.
+version_added: "2.5"
+requirements:
+    - hpOneView >= 4.0.0
+author:
+    - Felipe Bulsoni (@fgbulsoni)
+    - Thiago Miotto (@tmiotto)
+    - Adriane Cardozo (@adriane-cardozo)
+options:
+    state:
+        description:
+            - Indicates the desired state for the Logical Interconnect Group resource.
+              C(present) will ensure data properties are compliant with OneView.
+              C(absent) will remove the resource from OneView, if it exists.
+        default: present
+        choices: ['present', 'absent']
```

Comment:
These don't need to be quoted. We tend to quote only when it is is necessary in YAML.
Also, we tend to sort options in a list (if the order has no meaning).
So:
```yaml
        choices: [ absent, present ]
        default: present
```